stories/Overlays/Alert.stories.jsx

```diff
@@ -78,25 +79,28 @@ export const Sizes = () => {
           </div>
           <Alert
             isOpen={showAlertSmall}
-            message="Leaving this page will delete all unsaved changes."
+            message="All of your unsaved changes will be lost. This can't be undone"
```

Comment:
@nitinprojects _A Please add a period at the end of the message.

![Uploading Screenshot 2023-06-20 at 10.51.37 AM.png…]()